### PR TITLE
To be reactive the property needs to declared and initialized

### PR DESCRIPTION
**Pull request by bryan-gilbert**

### CODE REVIEW

**Review by WalterMoar — CHANGES_REQUESTED** [review]
Please remove the two console.log statements.

**Comment by WalterMoar** [comment]
Can you connect this to the issue?

File: ppr-ui/src/search/search-serial.ts [hunk 1]
```diff
@@ -39,7 +39,10 @@ export class SearcherSerial {
 
   private readonly _serialNumberRules: VFunction[] // ((v: string) => (string | boolean))[]
 
-  public constructor(baseUrl) {
+  public constructor(baseUrl: string) {
+    this._errorMessage = ''
+    this._results = []
+    this._term = ''
```
Comment (WalterMoar) [inline ppr-ui/src/search/search-serial.ts:45]: The label should be reactive as well.

Comment (bryan-gilbert) [reply ppr-ui/src/search/search-serial.ts:45]: the label contains a static value and does not exist as a private property.

Comment (WalterMoar) [reply ppr-ui/src/search/search-serial.ts:45]: Oh yeah - good point.

**Review by WalterMoar — APPROVED** [review]

**Pull request merged**